src/mbgl/text/placement.cpp

```diff
@@ -656,18 +661,17 @@ void Placement::placeBucket(const SymbolBucket& bucket,
         }
 
     } else {
-        for (const SymbolInstance& symbol : bucket.symbolInstances) {
-            placeSymbol(symbol);
+        auto endIt = bucket.symbolInstances.begin() + params.symbolInstanceEnd;
+        for (auto it = bucket.symbolInstances.begin() + params.symbolInstanceStart; it != endIt; it++) {
```

Comment:
styling nit: better have `auto beginIt = bucket.symbolInstances.begin() + params.symbolInstanceStar`

Reply:
nit: ++it